grafana/resource_cloud_stack.go

```diff
@@ -71,6 +76,12 @@ Changing region will destroy the existing stack and create a new one in the desi
 				Optional:    true,
 				Description: "Custom URL for the Grafana instance. Must have a CNAME setup to point to `.grafana.net` before creating the stack",
 			},
+			"wait_for_readiness": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Whether to wait for readiness of the stack after creating it. The check is a simple GET request to the stack URL (Grafana instance).",
```

Comment:
```suggestion
				Description: "Whether to wait for readiness of the stack after creating it. The check is a simple HEAD request to the stack URL (Grafana instance).",
```

If I read the implementation correctly, it is implemented using HEAD not GET.